Packs/ThreatConnect/Integrations/ThreatConnectV3/ThreatConnectV3.py

```diff
@@ -1607,6 +1708,444 @@ def add_group_tag(client: Client, args: dict):  # pragma: no cover
+                       'assets': include_assets,
+                       'associatedGroups': include_associated_groups,
+                       'include_all_metadata': include_all_metadata}.items() if
+                      should_include]
+    fields = set_fields(list_of_fields, is_victim_command=True)
+    filter = args.get('filter')
+    victim_id = args.get('victim_id')
+
+    limit = arg_to_number(args.get('limit')) or 50
+    page = arg_to_number(args.get('page')) or 0
+    page *= limit
+
+    url = VICTIM_API_PREFIX
+    if victim_id:
+        url += f'/{victim_id}'
+    url += f'?&resultStart={page}&resultLimit={limit}'
+    if fields:
+        url += f'{fields}'
+    if filter:
+        filter = urllib.parse.quote(filter.encode('utf8'))
+        url += f'&tql={filter}'
+    response = client.make_request(method=Method.GET, url_suffix=url)
+    outputs = response.get('data', {})
+    readable_output = tableToMarkdown('Victims', outputs, headers=['id', 'name', 'ownerName', 'description', 'org'])
+    return_results(CommandResults(
+        outputs_prefix='TC.Victim',
+        outputs_key_field='id',
+        outputs=outputs,
+        raw_response=response,
+        readable_output=readable_output,
+    ))
+
+
+def tc_create_victim_asset_command(client: Client, args: dict) -> None:
+    """
+    Creates a victim asset
+    Args:
+        client: ThreatConnect client
+        args: command arguments
+    """
+    victim_id = args.get('victim_id')
+    asset_type = AssetType(args.get('asset_type'))
+    asset_value = args.get('asset_value', '')
+    address_type = args.get('asset_address_type')
+    network_type = args.get('asset_network_type')
+    social_network = args.get('asset_social_network')
+    body = set_victim_asset(is_update=False,
+                            asset_type=asset_type,
+                            asset_value=asset_value,
+                            address_type=address_type,
+                            network_type=network_type,
+                            social_network=social_network)
+    body |= {'victimId': victim_id, 'type': asset_type}
+
+    response = client.make_request(method=Method.POST, url_suffix=VICTIM_ASSET_API_PREFIX, payload=json.dumps(body))
+    outputs = response.get('data', {})
+    readable_output = f'Victim Asset {outputs.get("id")} created successfully for victim id: {victim_id}'
+    return_results(CommandResults(
+        outputs_prefix='TC.VictimAsset',
+        outputs_key_field='id',
+        outputs=outputs,
+        raw_response=response,
+        readable_output=readable_output,
+    ))
+
+
+def tc_update_victim_asset_command(client: Client, args: dict):
+    """
+    Updates a victim asset
+    Args:
+        client: ThreatConnect client
+        args: command arguments
+    """
+    victim_asset_id = args.get('victim_asset_id')
+    # type is needed to determine which value should be sent in the request body
+    victim_asset = client.make_request(method=Method.GET,
```

Comment:
it's look like this API call are unnecessary,
as it's just to get the type, but as this are `update` and `is_update=True` no need to `type`

Reply:
Correct, but we need the type to later understand which key we should send in the request (if the type is *phone*, we need to send `phone: 111` and if the type is *address* we send `address: bla`)